cmd/skywire-cli/commands/rpc/root.go

```diff
@@ -0,0 +1,25 @@
+package clirpc
+
+import (
+	"net"
+	"time"
+
+	"github.com/skycoin/skywire-utilities/pkg/logging"
+	"github.com/skycoin/skywire/pkg/visor"
+)
+
+var (
+	// RPCAddr is the address to reach the rpc server
+	RPCAddr string
```

Comment:
Is this set via the persistent flag in the parent dir or where is this coming from? Does it need to be exported?